cmd/aperture-controller/controller.go

```diff
@@ -28,6 +30,7 @@ func main() {
 		controller.ModuleForControllerOTEL(),
 		fx.Provide(
 			clockwork.NewRealClock,
+			cache.Provide[selectors.ControlPointID],
```

Comment:
Why is Control Point cache required in the Controller?

Reply:
It's needed if we enable plugins

Reply:
You changed the ControlPointCache in Heartbeats to the new one, but did not provide it for controller.

Reply:
But Controller does not do Service Control Point discovery.

Perhaps we ca mark ControlPointCache as optional [here](https://github.com/fluxninja/aperture/blob/main/plugins/service/aperture-plugin-fluxninja/heartbeats/provide.go#L57) and do a nil check in Heartbeat [here](https://github.com/fluxninja/aperture/blob/main/plugins/service/aperture-plugin-fluxninja/heartbeats/heartbeats.go#L225)?

Reply:
Sure, we can, can you create an issue to track it?

Reply:
https://github.com/fluxninja/aperture/issues/1069